Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -210,9 +211,10 @@ def self.check_http_content(url, user_agents: [:default])
     "The URL #{url} could use HTTPS rather than HTTP"
   end
 
-  def self.http_content_headers_and_checksum(url, user_agent: :default)
+  def self.http_content_headers_and_checksum(url, hash_needed = false, user_agent: :default)
+    header_strategy = hash_needed ? "--include" : "-I"
```

Comment:
We should probably make "-I" into "--head" for clarity. Also, I'm not sure HEAD requests will work 100% of the time (e.g. on github). IIRC, the reason we're doing the full download every time is because it's more reliable.